README.md

```diff
@@ -445,6 +446,20 @@ Dodge et. al 2015. Tests Memory Networks on 4 tasks including reddit dialog task
 * [Reuters Corpora RCV2](http://trec.nist.gov/data/reuters/reuters.html)
 * [Spanish Billion words corpus with Word2Vec embeddings](http://crscardellino.me/SBWCE/)
 
+## NLP in Indic languages
+
+[Back to Top](#contents)
+
+### Hindi
+
+### Corpora and Treebanks
+
+* [Hindi Dependency Treebank](http://ltrc.iiit.ac.in/treebank_H2014/) - made by collaborative effort by five universities is multi-
+representational and multi-layered treebank for Hindi and Urdu.
```

Comment:
Single line explanation please? :) 

Being nitpicky so that you can get to a point where you can start correcting my mistakes 🥇